tests/testimpl/test_impl.go

```diff
@@ -0,0 +1,95 @@
+package common
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/arm"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/compute/armcompute"
+	"github.com/Azure/azure-sdk-for-go/sdk/resourcemanager/network/armnetwork/v5"
+	"github.com/gruntwork-io/terratest/modules/terraform"
+	"github.com/launchbynttdata/lcaf-component-terratest/types"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestComposableComplete(t *testing.T, ctx types.TestContext) {
+	subscriptionId := os.Getenv("ARM_SUBSCRIPTION_ID")
+	if subscriptionId == "" {
+		t.Fatalf("ARM_SUBSCRIPTION_ID must be set for acceptance tests")
+	}
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		t.Fatalf("Error getting credentials: %e\n", err)
+	}
+
+	computeClient, err := armcompute.NewVirtualMachinesClient(subscriptionId, cred, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	publicIPClient, err := armnetwork.NewPublicIPAddressesClient(subscriptionId, cred, nil)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	nicClient, err := armnetwork.NewInterfacesClient(subscriptionId, cred, nil)
+	if err != nil {
+		fmt.Println(err)
+	}
+
+	rgName := terraform.Output(t, ctx.TerratestTerraformOptions(), "resource_group_name")
+	expectedPublicIPId := terraform.Output(t, ctx.TerratestTerraformOptions(), "public_ip_id")
+	expectedPublicIPName := terraform.Output(t, ctx.TerratestTerraformOptions(), "public_ip_name")
+
+	bgCtx := context.Background()
+
+	t.Run("TestAlwaysSucceeds", func(t *testing.T) {
```

Comment:
Can we remove this?

Reply:
It's possible, but here are some concerns I have around that:

1. This "test" needs to exist in the Skeleton repository so that it has at least one test to run, so everything originating from that skeleton will have this test by default, which means we'd need to remember to remove it each time we used the skeleton. Adding more of these "manual human checks" to the PR process is undesirable in my opinion, we want to remove as much toil as we can from the process given the sheer volume of modules to manage and the expectation that reviewers may not be as consistently available going forward.

2. Removing this test from one repo implies we should be removing it from all repos where it's present and other tests exist. This effort would be semi-manual since I don't have a great way to pull in an AST of go files and selectively clip out matching tests, and would result in a lot of PR churn for arguable benefit.

3. Most of the elapsed test time is spent building and destroying TF resources, the presence of this "always succeeds" test contributes a negligible amount of time toward the compilation and execution phases. In other words, it's not really hurting anything by existing.

If you have some thoughts that guide us toward removing this I'm certainly open to hearing them, but given the above I don't see much reason to pursue this over anything else we have in flight.

Reply:
I dont have an idea of how to remove it using automation. I agree its not hurting to have leave it here but just doesn't look good. I always remove it after importing it from the skeleton. same way we remove the the `with_cake` example, rename the terratest function names and other changes in several places. Its more of an user training to remove irrelevant stuff after copying from skeleton. 

What I can think of is having a `TODO` in the comment section above this test in the skeleton. Having a TODO is flagged in most of the IDEs. That way the user can be alarmed to remove the TODO and hence this section.

Reply:
I'm on board with adding a `TODO` to the skeleton.

The next time I have to make a full pass through these repos to change files (probably not far out, TF registry is coming soon) I'll see if I can figure out a way to automate the removal, but no guarantees I'll be able to.